src/interfaces/cartItem.d.ts

```diff
@@ -0,0 +1,9 @@
+import { IDesign } from "@/interfaces/design";
+import { PriceBar } from "@/store/reducer/priceBarSlice";
+
+interface ICartItem extends IDesign {
+  quotation: string;
+  quotationDetails: PriceBar[];
+  // it's ok to be empty in this sprint
```

Comment:
Do you mean it's ok for previewPic to be an empty string for this sprint? But this is an interface, no need to comment on that for an interface.